ipython-profile.py

```diff
@@ -1,2 +1,3 @@
-from galaxy import get, put, get_galaxy_connection, _get_history_id
+from galaxy_ie_helpers import get, put
+from galaxy get_galaxy_connection, _get_history_id
```

Comment:
galaxy_ie can be imported one usegalaxy.org, indeed doing a `from galaxy_ie_helpers import get` does solve the problem. Will work on it as soon as I have reliable internet.